CLI.md

```diff
@@ -18,6 +18,7 @@ This document contains the help content for the `linera` command-line program.
 * [`linera sync-balance`↴](#linera-sync-balance)
 * [`linera sync`↴](#linera-sync)
 * [`linera process-inbox`↴](#linera-process-inbox)
+* [`linera query-new-validator`↴](#linera-query-new-validator)
```

Comment:
Why `new` ?

Reply:
Mmm ok you probably mean that they don't have to be validators yet, but is the distinction important?

Reply:
The command probably works with old/existing/new validators as long as they are connected, right?

Reply:
That's true, I'll rename it to `query-validator`.